custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
```

Comment:
Consider using template literals for string interpolation to improve readability and maintainability.

Suggested change:
```javascript
const introduction = {
  title: 'RAIA',
  text: `Software Developer • Not a Robot`,
};
```